dashboard/app/views/levels/_teacher_panel.html.haml

```diff
@@ -17,9 +17,10 @@
     - level_example_links = @level.examples.map do |example|
```

Comment:
where is `@level.examples` defined?

Reply:
Looks like this is at least somewhat handled by serialized properties. The details that I could find are here: https://github.com/code-dot-org/code-dot-org/pull/3748

Reply:
I think I am still missing something about how examples are even working at all.  Shouldn't `examples` be listed as a serialized property? If so, on which model class should they be listed? or, does the encrypted_ prefix do something special?

in the PR you linked to, `soft_buttons` is handled similarly to `examples`, but `soft_buttons` is declared as a serialized property.

Reply:
assuming `encrypted_examples` is where `examples` are stored, then this looks ok because I see that javalab has this: https://github.com/code-dot-org/code-dot-org/blob/7f6318869b3d1583a325b8cc0ee720e3b8cac19c/dashboard/app/models/levels/javalab.rb#L35